simpleclient_hibernate/src/main/java/io/prometheus/client/hibernate/HibernateStatisticsCollector.java

```diff
@@ -74,6 +83,19 @@ public HibernateStatisticsCollector add(SessionFactory sessionFactory, String na
     return this;
   }
 
+  /**
+   * Enables collection of per-query metrics. Produces a lot of monitoring data, so use with caution.
+   * <p>
+   * Per-query metrics have a label "query" with the actual HQL query as value. Parameters within
+   * the query will not be replaced (example: {@code select u from User u where id=?}).
```

Comment:
s/not//

Reply:
The sentence was ambiguous. Changed the comment to make it clear.